pom.xml

```diff
@@ -76,6 +77,23 @@
             <artifactId>mongobee</artifactId>
             <version>${mongobee.version}</version>
         </dependency>
+        <!-- rdf4j -->
+        <dependency>
+            <groupId>org.eclipse.rdf4j</groupId>
+            <artifactId>rdf4j-query</artifactId>
+            <version>${rdf4j.version}</version>
+        </dependency>
```

Comment:
Where is this dependency used? I see mostly `rio` classes being used.

Reply:
Removed